roles/openshift_control_plane/tasks/check_master_api_is_ready.yml

```diff
@@ -17,7 +17,7 @@
   when: openshift_apis is failed
 
 - debug:
-    msg: "{{ control_plane_logs_api.stdout_lines }}"
+    msg: "{{ control_plane_logs_api.stderr_lines }}"
```

Comment:
@mtnbikenc stderr?

Reply:
It depends on the tool that it being run under the hood.  I just noticed that the debug output was empty for stdout for the master-logs script.  All the output was in stderr, so I switched it.